lighthouse-cli/chrome-finder.ts

```diff
@@ -133,3 +182,27 @@ function canAccess(file: string): Boolean {
     return false;
   }
 }
+
+function uniq(arr: Array<any>) {
+    return arr.sort().filter((item: any, pos: number, _this: any) =>
+      !pos || item != _this[pos - 1]
+    );
+}
+
+function findChromeExecutables(desktopPath: string): Array<string> {
+  let installations: Array<string> = [];
+  if (canAccess(desktopPath)) {
+    let execPaths = execSync(`grep -ER "${chromeExecRegex}" ${desktopPath} | awk -F '=' '{print $2}'`)
+      .toString()
+      .split(newLineRegex)
+      .map((execPath: string) => execPath.replace(argumentsRegex, '$1'));
+
+    execPaths.forEach((execPath: string) => {
+      if (execPath && canAccess(execPath)) {
```

Comment:
will be able to simplify to

```js
canAccess(execPath) && installations.push(execPath);
```

and probably even one-line it in the arrow fn